### ISSUE TITLE
Print signal name when Test interface exits

### ISSUE THREAD

**Issue by ekrich** [scala-native/scala-native#2604]
The code in question should be here.
https://github.com/scala-native/scala-native/blob/63d07093f6d0a6e9de28cd8f9fb6bc1d6596c6ec/test-interface/src/main/scala/scala/scalanative/testinterface/signalhandling/SignalConfig.scala#L45-L71

Potential idea for a fix.
https://stackoverflow.com/questions/16509614/signal-number-to-name

**Comment by jmesyou** [comment]
#2722 conditionally fixes the issue for Unix. I plan to fixing the issue for Windows in a separate PR after #2722 is merged.
